core/iwasm/include/wasm_export.h

```diff
@@ -1826,6 +1836,12 @@ wasm_runtime_get_custom_section(const wasm_module_t module_comm,
 WASM_RUNTIME_API_EXTERN void
 wasm_runtime_get_version(uint32_t *major, uint32_t *minor, uint32_t *patch);
 
+/**
+ * Get max supported version for a package type
+ */
+WASM_RUNTIME_API_EXTERN unsigned int
+wasm_runtime_get_max_package_version(package_type_t package_type);
```

Comment:
How about `uint32 get_package_current_version(package_type_t package_type)`?

Reply:
I think this is the same type of naming decision as #3600, so I'll just refer to my comment on that issue:

https://github.com/bytecodealliance/wasm-micro-runtime/pull/3600#discussion_r1668790429

And whatever is decided there, we should probably also do here.